install-components.md

```diff
@@ -127,71 +125,66 @@ that you plan to create the `Workload` in:
     apiVersion: v1
     kind: ServiceAccount
     metadata:
-      name: default
+      name: supplychain
```

Comment:
*issue*: `supplychain` isn't the right name for this SA as it's tied to a specific `Workload` (and a `Delivery` in this documentation).

Reply:
I can switch this back to `default` if we want; long-term it seems like we might want to separate "what identity does the supply chain run as" and "what identity does the deployed workload run as". With the proposed RoleBindings, we'd also need to add `ImagePullSecrets` for the "identity the deployed workload runs as", so it may be reasonable to separate these for now.

Reply:
(Switched back to `default` for now)